var/log/phpcs.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
Looks like contents of the `var` directory have been committed somehow. Please remove the newly-committed files and figure out if something was broken to allow them to get committed in the first place.